src/v3/src/mocks/scenario/development.ts

```diff
@@ -13,7 +13,7 @@
 import { scenario } from '../registry';
 
 const getRouteUrl = (url: string) => {
-  const parts = url.split('/idp/idx/', 1);
+  const parts = url.split('/idp/idx/');
```

Comment:
hopefully there's no difference

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split#parameters

when limit is set as 1, it only keeps i item in array, need to change it back to fix e2e failures